pkg/api/types.go

```diff
@@ -3964,6 +3964,14 @@ const (
 	ResourceLimitsEphemeralStorage ResourceName = "limits.ephemeral-storage"
 )
 
+// The following identify resource prefix for Kubernetes object types
+const (
+	// HugePages request, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
+	ResourceRequestsHugePagesPrefix = "requests.hugepages-"
```

Comment:
we do not support overcommit on hugepages, so can we just quota requests and ignore limits for now?

Reply:
i am trying to think how this would work when we have a resource discovery api, and we say 'this resource does not support burst', in which case, we would most likely only quota the request, and ignore the limit.